ambry-api/src/main/java/com.github.ambry/account/AccountService.java

```diff
@@ -87,4 +88,19 @@
    * @return A collection of {@link Account}s.
    */
   public Collection<Account> getAllAccounts();
+
+  /**
+   * Adds a {@link Consumer} for newly created or updated {@link Account}s.
+   * @param accountUpdateConsumer The {@link Consumer} to add.
+   * @return {@code true} if the specified {@link Consumer} has already been added, {@code false}
```

Comment:
"has already been added"? Does that mean this will return `false` the first time?